sample.coffee

```diff
@@ -19,14 +19,17 @@ myData = FileCollection({
 
 if Meteor.isClient
 
+   # This assigns a file drop zone to the "file table"
+   # once DOM is ready so jQuery can see it
+   Template.collTest.onRendered ->
```

Comment:
@Digital-Thor  Sorry for letting this drop. I agree that a function successfully registered with `.onRendered()` can't possibly be called before the DOM is ready, because it is called when a template *instance* is inserted in the DOM.

My concern was that a template *object* such as `Template.collTest` may not be guaranteed to exist until a `Meteor.startup` callback is invoked. So prematurely calling `Template.collTest.onRendered()` could fail and throw. 

However, the Meteor documentation on "load order" answers this definitively:

> HTML template files are always loaded before everything else

 http://docs.meteor.com/#/full/fileloadorder

So this looks good